lib/person.rb

```diff
@@ -0,0 +1,27 @@
+class Person
+  attr_reader :name,
+              :interests,
+              :supplies
+
+  def initialize(attributes)
+    @name      = attributes[:name]
+    @interests = attributes[:interests]
+    @supplies  = Hash.new {|hash, key| hash[key] = 0}
```

Comment:
```suggestion
    @supplies  = Hash.new(0)
```

or 
```suggestion
    @supplies  = {}
```

is a bit simplier